crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -230,8 +230,8 @@ x.....l.cc+ccccccccccc..l....x
 x.....l.cc..........cc..l....x
 x.....l.ccc.........cc..l....x
 x.....ll.cc.....2.3.cc.ll....x
```

Comment:
```suggestion
x.....ll.cc.....G.G.cc.ll....x
```
This vault is one we wouldn't accept these days (big spoiler and has monsters). Hit the remove button on this, or if you don't think that's appropriate turn the 2 and 3 into statues and remove the rat and oozes and put some gaps in the lava.